package.json

```diff
@@ -18,37 +17,43 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:e2e": "cucumber-js -p default"
   },
   "dependencies": {
-    "@nestjs/common": "^7.5.1",
-    "@nestjs/core": "^7.5.1",
-    "@nestjs/platform-express": "^7.5.1",
+    "@nestjs/common": "^7.6.13",
+    "@nestjs/core": "^7.6.13",
```

Comment:
Version bumps should be in its own PR in the future

Reply:
There was some dependencies issue, a few of the dependencies were not working with cucumber, so I had to change the versions.

Reply:
ah ok 👍